A32NX/ModelBehaviorDefs/A32NX/FMC.xml

```diff
@@ -0,0 +1,860 @@
+<<<<<<< HEAD
```

Comment:
Just in case you didn't notice, you've committed a number of merge conflict markers in a few different files here.

Reply:
yeah this was a mistake when workin from a different machine, fixed here 8e489e3